docs/modules/ROOT/pages/explanations/decisions/mongodb.adoc

```diff
@@ -0,0 +1,39 @@
+= MongoDB
+
+== Problem
+
+We want to Provide MongoDB via AppCat.
+But their licensing might be an issue.
+This decision looks closely at the licensing and alternative products that can be used as drop-in replacements for MongoDB.
+
+== Proposals
+
+Proposal 1: Use MongoDB Community Edition::
+MongoDB community edition is licensed under the https://github.com/mongodb/mongo/blob/master/LICENSE-Community.txt[SSPL].
+The SSPL dictates that the source code for the tooling that provides the services, needs to be available to the public as well.
```

Comment:
```suggestion
The SSPL dictates that the source code for the tooling that provides the services needs to be available to the public.
```

Reply:
It doesn't only dictated that the source is available, form what I've gathered the source then also needs to be the SSPL license.